packages/dcos-integration-test/extra/test_metrics.py

```diff
@@ -496,3 +502,15 @@ def get_app_metrics(dcos_api_session, node: str, container_id: str):
     assert 'datapoints' in app_metrics, 'got {}'.format(app_metrics)
     assert 'dimensions' in app_metrics, 'got {}'.format(app_metrics)
     return app_metrics
+
+
+@retrying.retry(stop_max_delay=(30 * 1000))
+def assert_metric_reported(dcos_api_session, master: str, metric_substring: str):
```

Comment:
A [similar helper defined here](https://github.com/dcos/dcos/pull/3744/files#diff-a99db445cccf205fe11239e4be6de2aaR48) can be used instead

Reply:
Too many trees; I could not see the forest :+1: